src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/SlimefunStartupTask.java

```diff
@@ -56,6 +57,9 @@ public void run() {
                 SlimefunPlugin.logger().log(Level.SEVERE, x, () -> "An Error occurred while trying to load World \"" + world.getName() + "\" for Slimefun v" + SlimefunPlugin.getVersion());
             }
         }
+        
+        // Load/Unload Worlds, only after all plugins have started up.
```

Comment:
Add a "Fixes #2862" to this so we know in the future why it's here

```suggestion
        // Load/Unload Worlds, only after all plugins have started up. Fixes #2862 
```